src/xc_integrator/replicated/host/reference_replicated_xc_host_integrator_exc_grad.hpp

```diff
@@ -149,19 +165,43 @@ void ReferenceReplicatedXCHostIntegrator<ValueType>::
     auto* vrho       = host_data.vrho.data();
     auto* vgamma     = host_data.vgamma.data();
 
+    auto* tau        = host_data.tau.data();
```

Comment:
Until we can check this, I would probably opt to disable this code path - the last thing we need is having something be accessible just to find out it doesn't work. We can discuss offline if there's an easy way to check this implementation (compare with NWChem, etc)